masking/semantic-type.json

```diff
@@ -3,12 +3,41 @@
   "value": {
     "semanticTypeSettingValue": {
       "types": [
+        {
+          "id": "bb.default",
+          "title": "Default",
```

Comment:
Do we need this title/description for built-in types?

Reply:
We don't need them, but I think it's good for the sample

Reply:
SGTM.